stdlib/contrib/mhall119/helpers/helpers.flux

```diff
@@ -0,0 +1,94 @@
+// Package helpers provides shortcut functions for small common tasks
+//
+// ## Metadata
+// introduced: ??
+package helpers
+
+import "array"
+import "json"
+// yieldValue lets you display a scalar in the InfluxDB UI by
+// wrapping it in an Array
+//
+// ## Parameters
+// - v: Input data.
+// - name: Name to pass to the subsequent yield() call.  Default is 'value'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldValue(v: testData.testValue)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldValName="value"
+yieldValue = (v, name=defaultYieldValName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: v},
+      ]
+    )
+    |> yield(name: name)
+
+// yieldObject lets you display an object in the InfluxDB UI by
+// encoding it as String
+//
+// ## Parameters
+// - o: Object to yield
+// - name: Name to pass to the subsequent yield() call.  Default is 'object'.
+//
+// ## Examples
+//
+// ### Yield an option value
+// ```no_run
+// import "contrib/mhall119/helpers"
+//
+// option testData = {testValue: 125}
+// yieldObject(v: testData)
+// ```
+//
+// ## Metadata
+// tags: 
+//
+defaultYieldObjName="object"
+yieldObject = (o, name=defaultYieldObjName) =>
+    array.from(
+      rows: [
+          {_time: now(), _value: display(v: o)},
+      ]
+    )
+    |> yield(name: name)
+
+// yieldObject lets you display an object in the InfluxDB UI by
+// encoding it as a JSON string and wrapping it in an Array
```

Comment:
Wrong function name.

I'm not sure I fully understand the utility of this function. What's the use case? This seems like a round about way to do the same thing as `yieldRecord`.

```suggestion
// yieldJSON returns a stream of tables containing a JSON string.
// This makes it possible to display JSON in an InfluxDB visualization.
```